Fix warnings

evgeniiz321 opened the pull request.

roman-khimov commented on pytest_tests/tests/services/rest_gate/test_rest_gate.py:
```diff
@@ -7,19 +7,19 @@
 from helpers.wellknown_acl import PUBLIC_ACL
-from rest_gw.rest_utils import get_object_and_verify_hashes
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from rest_gw.rest_utils import get_object_and_verify_hashes
```
Formatting change, not related to marks.

evgeniiz321 replied:
removed

roman-khimov commented on pytest_tests/tests/services/rest_gate/test_rest_gate_new.py:
```diff
@@ -8,19 +8,19 @@
 from helpers.wellknown_acl import PUBLIC_ACL
-from rest_gw.rest_utils import get_object_and_verify_hashes
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from rest_gw.rest_utils import get_object_and_verify_hashes
```
Here as well.

evgeniiz321 replied:
removed

roman-khimov reviewed (commented):
How come 5f5f72a313520452fc46fff5f7705accffc64aba commits a `warp` binary?

Can e have a bit more details for 5de9dabcba19070a0798ca6cb1e9531bff161a28?

Where are references to original issue?

evgeniiz321 commented:
> How come [5f5f72a](https://github.com/nspcc-dev/neofs-testcases/commit/5f5f72a313520452fc46fff5f7705accffc64aba) commits a `warp` binary?
> 
> Can e have a bit more details for [5de9dab](https://github.com/nspcc-dev/neofs-testcases/commit/5de9dabcba19070a0798ca6cb1e9531bff161a28)?
> 
> Where are references to original issue?

Removed warp binary, was there by accident.
Added a little bit more details to the commit with an invalid escape sequence.
Added a reference to the new issue with warnings.
